texts/navigation.yml

```diff
@@ -0,0 +1,10 @@
+title: Navigation
```

Comment:
This is not used you can remove it, can't you?

Reply:
It's not used yet. It should be screen-reader only heading. Thank you